packages/hash/frontend/src/pages/[account-slug]/entities/[entity-uuid].page/entity-editor/properties-section/property-table/cells/value-cell/value-cell-editor/array-editor/sortable-row.tsx

```diff
@@ -111,7 +112,12 @@ export const SortableRow = ({
           }}
         />
       ) : (
-        <ValueChip value={value} selected={selected} />
+        <ValueChip
+          value={value}
+          selected={selected}
+          icon={{ icon: faText }}
+          tooltip="Text"
```

Comment:
I expect this icon and tooltip text to change according to data type, right? Is this being left out for a future PR?

Reply:
That has it's own ticket on Asana, but thanks!